<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>603</width>
    <height>648</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sincro</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="2,0,0,8">
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="sizeConstraint">
         <enum>QLayout::SetMinAndMaxSize</enum>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,9,1">
          <property name="sizeConstraint">
           <enum>QLayout::SetFixedSize</enum>
          </property>
          <item>
           <widget class="QLabel" name="label">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>0</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Calibri</family>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Master</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="EtopM">
            <property name="font">
             <font>
              <family>Calibri</family>
              <pointsize>10</pointsize>
             </font>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="BSelM">
            <property name="font">
             <font>
              <family>Calibri</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Canviar</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,9,1">
          <property name="sizeConstraint">
           <enum>QLayout::SetFixedSize</enum>
          </property>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>0</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Calibri</family>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="lineWidth">
             <number>0</number>
            </property>
            <property name="text">
             <string>Slave</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="EtopS">
            <property name="font">
             <font>
              <family>Calibri</family>
              <pointsize>10</pointsize>
             </font>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="BSelS">
            <property name="font">
             <font>
              <family>Calibri</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Canviar</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QPushButton" name="BRev">
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Revisar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="BSinc">
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Sincronitzar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="LReady">
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="BInvertirMS">
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>M &lt;-&gt; S</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableWidget" name="TAccions">
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="columnCount">
       <number>4</number>
      </property>
      <column/>
      <column/>
      <column/>
      <column/>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
